### DISCUSSION TITLE
Give your feedback on odo

### DISCUSSION THREAD

**Discussion by serenamarie125** [redhat-developer/odo#4642] [General]
👂🏼 We would like to hear your feedback (positive and negative) on odo overall. 

We will consider all feedback when planning improvements in later releases as we are vested in focusing on areas where the greatest improvements can be made for users.

Please provide whatever feedback you have here in this discussion, or suggest other ways to solicit more feedback!

**Comment by erikbgithub** [comment]
I did this free course and loved using odo: https://learn.openshift.com/introduction/developing-with-odo/

I love:
 * the output is pretty, readable and informative (utf8 and colors <3)
 * the interactive `watch` mode in the background
 * that the interface motivates you to think `project -> catalog -> component -> config -> push`
 * the arguments altogether are slightly more readable as the comparable `oc` alternatives

Could be better:
 * honestly haven't experienced anything yet that I could complain about, and I usually love complaining.

**Reply by serenamarie125** [reply]
Thanks for the feedback @erikbgithub !!
